features/rhelah/atomic/rpm-ostree_bad_ca_cert.feature

```diff
@@ -0,0 +1,16 @@
+Feature: tests whether RH CA pinning works for rpm-ostree
+    It is critical for security that we're "pinned" to the RHN CA cert. If another server certificate is presented we should error out with something like Unacceptable TLS certificate
+
+Scenario: 0. Host provisioned and subscribed
+	When "all" host is auto-subscribed to "prod"
+	Then subscription status is ok on "all"
+and "1" entitlement is consumed on "all"
+
+Scenario: 1. Prepare mitmproxy
+	Given "git" are already installed on "rhel7"
+	and run command "command git clone https://github.com/ljozsa/tools.git" on "rhel7"
```

Comment:
Would be nice to have this more generic, e.g. "checkout the git repo "<repo>" on "<host>". That way it would be easier to read and also re-usable elsewhere.